src/config.ts

```diff
@@ -31,6 +33,13 @@ class ClientConfig {
    */
   private _fetch: FetchFunction | null = tryGettingFetch();
 
+  /**
+   * Number of tiles to keep in cache
+   */
+  public cacheSize: number = 200;
```

Comment:
I purposefully chose a generic name because it could (in the future) be used for other kinds of tiles, even though right now it's only for terrain tiles.

Reply:
Oh, I just realized this is a property on the `ClientConfig`, so it will not be automatically visible on the `SdkConfig`, so it's not as confusing as a I feared.

I was afraid of people doing:
```js
const map = new Map({
  cacheSize: 100000
});
```
and expecting it to have some kind of effect on map performance.

So maybe just `tileCacheSize` ? I don't think we are ever going to reuse the cache for anything else than tiles, right? Can't imagine a scenario of storing more item types in a single cache..

Reply:
Modified to `tileCacheSize` 👌